.github/workflows/e2e-report.yml

```diff
@@ -27,12 +35,15 @@ jobs:
             E2E_RUN_ID=$(gh run list -w test-e2e.yml -e schedule -s success --json databaseId --jq ".[0].databaseId" --repo $GITHUB_REPOSITORY)
           fi
           echo "runId=$E2E_RUN_ID" >> $GITHUB_OUTPUT
-      - name: Download latest e2e results
+      - name: Download e2e results
         if: ${{ steps.get-run-id.outputs.runId }}
         run: |
-          echo "Downloading latest test results from run https://github.com/netlify/next-runtime/actions/runs/${{ steps.get-run-id.outputs.runId }}"
-          rm e2e-report/data/test-results.json
-          gh run download ${{ steps.get-run-id.outputs.runId }} -n "latest-test-results.json" -D e2e-report/data/ --repo $GITHUB_REPOSITORY
+          version="${{ inputs.version }}"
+          version=${version:-latest}
```

Comment:
we need a second layer of defaulting here because the scheduled run won't have the workflow_dispatch default populated

Reply:
True, but we can have it manual only just so we can do first release, and then fix this.
I don't think we're ready to automatically deploy on a schedule yet, in terms of test stability. Would be great to have a deploy preview automatically though (so then it's much easier to evaluate before manual release)

Reply:
To clarify for anyone reading this: this job already runs on a schedule and only ever creates deploy previews. We publish deploys to production manually in the Netlify UI.